Utils/Dataflow/009_oracleConnector/README

```diff
@@ -24,37 +24,6 @@ To run `prodsys2ES` query only, use PLAIN mode:
 
   ./oracle2JSON.py --config $CONFIG --mode PLAIN
 
-The configuration file:
-```
-[oracle]
-dsn: USER/PASSWORD@HOST:PORT/SERVICE_NAME
-
-[queries]
-# relative path to tasks query
-tasks: query/prodsys2ES.sql
-# relative path to datasets query
-datasets: query/datasets.sql
-
-[process]
-# Query response processing mode:
-#  PLAIN   -- pass records to output independently of each other
-#  SQUASH  -- link 'datasets' records to 'task', joining them into a single
-#             record
-mode = SQUASH
-
-[timestamps]
-# initial timestamp
-initial = 04-03-2014 00:00:00
-# final timestamp
-final = 10-05-2016 00:00:00
-# hours offset from current timestamp
-step = 24h
-
-[logging]
-# Current offset file
-# (relative to the dir with the config file or absolute path)
-# offset_file = .offset
-```
-
-Current offset by default is stored in the same directory as the config file,
-  but it can be configured.
+A template for the configuration file can be obtained in Dataflow's corresponding directory:
+
+  dkb/Utils/Dataflow/config/009.cfg.example
```

Comment:
I'd rather use git's notation for paths within repository: `Utils/Dataflow/config/009.cfg.example`, just as `git` shows when asked about full names:
```
$ pwd
/home/mgolosova/work/DKB/git/Utils/Dataflow/config
$ git ls-files --full-name
Utils/Dataflow/config/009.cfg.example
Utils/Dataflow/config/095.cfg.example
Utils/Dataflow/config/README
```

And about formatting -- could you pleas:
* wrap the text at 80 characters;
* add newline at the end of file (see red symbol at the end of last line in the GitHub representation)?

Reply:
Sure, will do.

Reply:
Done.

Reply:
Thank you!